### PR TITLE
Added installation instructions for "Forms & HTML"

### PR DESCRIPTION
**Pull request by DfKimera**
The Forms/Html helpers are no longer bundled in the core, but shipped in a separate package.
The installation instructions for this package are missing, as well as the information that the helpers are no longer bundled in the core.

### CODE REVIEW

File: html.md [hunk 1]
```diff
@@ -13,6 +14,32 @@
...
 
+<a name="installation"></a>
+## Installation
+
+The Forms & HTML helpers are no longer bundled with Laravel core. 
+
+You can install this package through Composer. Edit your project's `composer.json` file to require `laravel/html`.
+
+    "require": {
+        "laravel/html": "~5.0"
```
Comment (GrahamCampbell) [inline html.md]: Not correct,

**Comment by taylorotwell** [comment]
These documentation pages aren't displayed for the 5.0 docs.

**Comment by taylorotwell** [comment]
Also this information is in the change log.

**Comment by DfKimera** [comment]
If it no longer belongs in the docs, shouldn't the page be removed instead? It still shows up on Google results.

**Comment by adamgoose** [comment]
Live Documentation here: http://laravelcollective.com/docs/5.0/html

**Comment by taylorotwell** [comment]
Removed it just now.​

**Pull request closed without merging**